samples/meetings-live-code-interview/csharp/README.md

```diff
@@ -60,7 +60,7 @@ sequenceDiagram
 - [Ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
```

Comment:
We can change header/title of readme file as
"This sample demos a live coding in a teams meeting stage using live share SDK."

Also, can add doc links of live share sdk in further reading section.